app/src/main/java/com/cornellappdev/score/components/UpcomingGameCard.kt

```diff
@@ -0,0 +1,161 @@
+package com.cornellappdev.score.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.score.theme.poppinsFamily
+import com.example.score.R
+
+@Composable
+fun UpcomingGameHeader(
+    leftTeamLogo: Painter,
+    rightTeamLogo: Painter,
+    gradientColor1: Color,
+    gradientColor2: Color ,
+    modifier: Modifier
+) {
```

Comment:
Make sure to autoformat. You can autoformat the whole codebase by right clicking some root package and hitting `Reformat Code`

Reply:
Also, are we expecting this header to be used anywhere else besides the card? If not, we can make this a private function.

But I suppose this might be reused on the game detail page or something like that. If that's ur vision, feel free to keep this as public.